### PR TITLE
Removes cross-links given shortness of document

### LINKED ISSUE #266: Remove links to tips
There are quite a few tips with these cross references. As I'm reading I do find them a bit odd for this type for a manuscript that is not _that_ long. I'd probably remove them throughout, unless someone has a strong case for keeping them.

_Originally posted by @cgreene in https://github.com/Benjamin-Lee/deep-rules/pull/258#discussion_r503165828_

### PR DESCRIPTION
**Pull request by fmaguire**
Closes #266

And provides a draft figure for #261

### CODE REVIEW

**Review by rasbt — CHANGES_REQUESTED** [review]
The figure is really awesome!

Below are just some suggested tweaks to the text:

Since we use AE in the rest of the text, changing the following would be good for consistency:

6) optimisiation -> optimization
7) regularise -> regularize

We also decided to remove 2nd person everywhere. So, below are some suggestion (but there may be better ways to reword those instances)

2) your DL model -> a DL model
4) peculiarities of your data -> peculiarities of the data
8) why your model -> why a model

Lastly, maybe the following change would make it sound a tad letter in 9)

9) derived from model -> derived from a model

**Comment by Benjamin-Lee** [comment]
Is it standard to have a figure before the introduction heading? Or should it go immediately after?

**Comment by Benjamin-Lee** [comment]
Also, this figure is 100% what we should be using as a thumbnail: https://github.com/Benjamin-Lee/deep-rules/blob/master/thumbnail.png

Can you update that file as well?

**Comment by rasbt** [comment]
> Is it standard to have a figure before the introduction heading? Or should it go immediately after?

Some papers have so-called "graphical abstracts" which are essentially figures that go next/below the abstract. Otherwise it is not standard. My Ph.D. advisor was pretty strict about that: she never allowed a figure to appear before it is referenced in the text. I think most people are more relaxed about that these days but it would maybe be weird to put it above the heading if it is not a graphical abstract.

**Comment by fmaguire** [comment]
A graphical abstract was kinda my thinking but happy to move it into the introduction.

I'm knee deep in the latest covid variant stuff but will try to get that updated and thumbnail added ASAP

**Comment by Benjamin-Lee** [comment]
No rush at all. Thanks for the great work!

**Comment by rasbt** [comment]
> A graphical abstract was kinda my thinking but happy to move it into the introduction.

I think that would be great, we just have to make sure whether the journal supports that, @Benjamin-Lee do you know more about this?

**Comment by Benjamin-Lee** [comment]
I've never seen it at PLOS Comp Bio, only figure 1 pseudo-graphical abstracts. This is their [page on figures](https://journals.plos.org/ploscompbiol/s/figures) but it clearly either doesn't apply to Ten Simple Rules articles or isn't enforced. Specifically, it says not to use article title (violated in the antiracist lab paper) and to use Times/Helvetica font (violated by PE Bourne himself).

**Comment by fmaguire** [comment]
Updated as per issue discussion.

File: content/02.intro.md [hunk 1]
```diff
@@ -21,6 +22,8 @@ To make deep learning more accessible to biological researchers, we solicited in
 These individuals collaboratively contributed to this manuscript's writing using the GitHub version control platform [@url:https://github.com/Benjamin-Lee/deep-rules] and the Manubot manuscript generation toolset [@url:https://greenelab.github.io/meta-review/].
 The goal was to articulate a practical, accessible, and concise set of guidelines and suggestions for biologically oriented researchers to follow when using deep learning.
```
Comment (agitter) [inline content/02.intro.md:23]:
```suggestion
The goal was to articulate a practical, accessible, and concise set of guidelines and suggestions for biologically oriented researchers to follow when using deep learning (Figure @fig:overview-fig).
```

This figure placement looks good to me, but I think we need to reference it in the text as well.

Comment (fmaguire) [reply content/02.intro.md]: Good shout, done.

**Review by rasbt — APPROVED** [review]

**Review by agitter — APPROVED** [review]
It looks great in the PDF artifact

**Pull request merged**